extensions/amp-story/1.0/amp-story-draggable-drawer.js

```diff
@@ -159,6 +159,17 @@ export class DraggableDrawer extends AMP.BaseElement {
       this.containerEl_.insertBefore(spacerEl, this.contentEl_);
       this.contentEl_.appendChild(headerShadowRootEl);
       this.element.classList.add('amp-story-page-attachment-ui-v2');
+      this.headerEl_.classList.add('amp-story-page-attachment-ui-v2');
+      const handler = (entries) => {
+        this.headerEl_.classList.toggle(
+          'i-amphtml-story-draggable-drawer-header-stuck',
+          !entries[0].isIntersecting
+        );
+      };
+      const observer = new window.IntersectionObserver(handler);
+      setTimeout(() => {
+        observer.observe(spacerEl);
+      }, 1000);
```

Comment:
Why is there a need to wait 1s to install the observer? (might be worth adding a comment)